onig_sys/build.rs

```diff
@@ -92,21 +95,14 @@ fn compile(link_type: LinkType) {
     };
 
     // Execute the oniguruma NMAKE command for the chosen architecture.
-    let cmd = format!("make_win{}.bat", bitness);
-    println!("{}", cmd);
-    let r = Command::new("cmd")
-        .args(&["/c", &(onig_sys_dir.join(cmd).to_string_lossy())])
-        .current_dir(&build_dir)
-        .env_remove("MFLAGS")
```

Comment:
Need to make sure that these are removed or the build will fail on some configurations. NMAKE is not make!